log/gps/gps_destinator_trk.ksy

```diff
@@ -0,0 +1,82 @@
+meta:
+  id: gps_destinator_trk
+  title: Destinator track
+  application: Destinator
+  file-extension:
+    - dat
+  license: GPL-2.0-or-later
+  encoding: utf-16le
+  endian: le
+
+doc: |
+  Destinator formats
+
+doc-ref:
+  - https://web.archive.org/web/20160310115940/http://mozoft.com/d3log.html
+  - https://web.archive.org/web/20071011022251/http://www.destinatortechnologies.net/uk/products/products/bundles/
+  - https://www.gpsbabel.org/htmldoc-development/fmt_destinator_trl.html
+  - https://github.com/gpsbabel/gpsbabel/blob/e1e195beca8e0f24ca0e41c02d1986af13865cd0/destinator.cc#L264L326
+
+
+seq:
+  - id: points
+    type: point
+    repeat: eos
+
+types:
+  point:
+    doc: |
+      152 bytes
+      For 3.0.75_103, PN 5.1.80 and 6.0.0.24556
+    seq:
+      - id: lon
+        type: f8
+      - id: lat
+        type: f8
+      - id: alt
+        type: f8
+      - id: unkn0
+        type: f8
+      - id: unkn1
+        type: f8
+      - id: unkn2
+        type: f8
+      - id: fix
+        type: u4
+        enum: fix
+      - id: satellites_visible
+        type: u4
+      - id: satellites_info
+        type: u4
+        repeat: expr
+        repeat-expr: 12
+        doc: 0 for unused channels.
+      - id: date
+        type: u4
+        doc: DDMMYY
+      - id: milliseconds
+        type: f4
+        -unit: 0.001 s
+        doc: "UTC decimal: HHMMSSsss"
```

Comment:
This seems contradictory. The `id` and `-unit` make it sound like it's just a number of milliseconds (since midnight?), so for example 00:01:23.456 would be stored as 83456.0. But the `doc` makes it sound like the number needs to be interpreted as decimal (like the `date` field), so for example 00:01:23.456 would be stored as 123.456 (or 123456.0?).